docs/plot.rst

```diff
@@ -213,6 +216,25 @@ Price
       :file: img/single/lmp_usa_map.html
 
 
+  - plot capacity vs cost curve slope of generators in an area (`notebook
+    <https://github.com/Breakthrough-Energy/PostREISE/blob/develop/postreise/plot/demo/scatter_capacity_vs_cost_curve_slope_demo.ipynb>`__)
+
+    .. code-block:: python
+
+        from powersimdata.scenario.scenario import Scenario
+        from powersimdata.utility.helpers import PrintManager
+        from postreise.plot.plot_scatter_capacity_vs_cost_curve_slope import (
+            plot_scatter_capacity_vs_cost_curve_slope
+        )
```

Comment:
Should we follow consistent flake8 import format in these code snippets?

Reply:
Ah, the newline between the powersimdata import and the postreise one

Reply:
Something like a blank line between import of other packages and the current one (as we did above).

Reply:
Done in 575337abf9e986ded5c68cd33e895247f67c6520. What needed to be done:
* organize statements alphabetically
* separate postreise modules and others with a blank line in between
* use 2 blank lines between imports and code